### PR TITLE
[Rule Tuning] Change default time query for rounding days

### PR DESCRIPTION
**Pull request by nkhristinin**
To improve cache performance for the IM rule we have ideas to round the timestamp query by days.

From @timestamp >= "now-30d" - to @timestamp >= "now-30d/d"

### CODE REVIEW

**Comment by w0rk3r** [comment]
(`updated_date` needs to be updated)

**Review by ecezalp — APPROVED** [review]
LGTM

File: rules/cross-platform/threat_intel_filebeat7x.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [metadata]
 creation_date = "2021/04/21"
 maturity = "production"
-updated_date = "2021/10/29"
+updated_date = "2022/01/26"
```
Comment (brokensound77) [inline rules/cross-platform/threat_intel_filebeat7x.toml:4]: This rule was deprecated for 8.0+ #1703, which is why there is a conflict. You can revert the changes on this file to resolve the conflict.

Then, in a separate PR targeting 7.16, this file can be changed with this update and then backported to 7.14-7.15 (and maybe 7.13, depending on when the rule was introduced)

**Review by brokensound77 — CHANGES_REQUESTED** [review]
https://github.com/elastic/detection-rules/pull/1713/files#r793134842

File: rules/cross-platform/threat_intel_filebeat7x.toml (hunk 1 above)
Comment (brokensound77) [reply rules/cross-platform/threat_intel_filebeat7x.toml:4]: for more context, you can see:
* 8.0 (and main) [deprecated](https://github.com/elastic/detection-rules/blob/8.0/rules/_deprecated/threat_intel_filebeat7x.toml)
* 7.13 (-7.16) [production](https://github.com/elastic/detection-rules/tree/7.13/rules/cross-platform)

**Review by brokensound77 — APPROVED** [review]
Thanks for the changes - LGTM 👍

**Pull request merged**